geoportal/c2cgeoportal_geoportal/__init__.py

```diff
@@ -325,13 +325,15 @@ def includeme(config: pyramid.config.Configurator):
     get_user_from_request = create_get_user_from_request(settings)
     config.add_request_method(get_user_from_request, name="user", property=True)
     config.add_request_method(get_user_from_request, name="get_user")
-    # Be able for an organisation to override the method to use alternate roles name for
-    # the standard roles 'anonymous', 'registred' and 'intranet'.
+    # Be able for an organisation to override the method to use alternate:
+    # Roles name for the standard roles 'anonymous', 'registred' and 'intranet'.
     config.add_request_method(lambda request, role_type: role_type, name="get_organization_role")
-    # Same for Print URL
+    # Print URL
     config.add_request_method(
         lambda request: request.registry.settings["print_url"], name="get_organization_print_url"
     )
+    # Interface name (in the config and in the admin interface)
+    config.add_request_method(lambda request, interface: interface, name="get_organization_interface")
```

Comment:
So far in the documentation and in the code as far as I can see, we write "organisation" (not "organization"). We should stay consistent or it will be hard to debug.

Reply:
@jwkaltz Finally we decide to use the US version with a 'z' everywhere.

Reply:
> @jwkaltz Finally we decide to use the US version with a 'z' everywhere.

But not in the PR or in the branch name ;)